libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/components/ClickableLinkText.kt

```diff
@@ -40,14 +41,15 @@ import io.element.android.libraries.designsystem.theme.components.Text
 import kotlinx.collections.immutable.ImmutableMap
 import kotlinx.collections.immutable.persistentMapOf
 
+@OptIn(ExperimentalTextApi::class)
 @Composable
 fun ClickableLinkText(
     text: AnnotatedString,
-    linkAnnotationTag: String,
-    onClick: () -> Unit,
-    onLongClick: () -> Unit,
     interactionSource: MutableInteractionSource,
     modifier: Modifier = Modifier,
+    linkAnnotationTag: String = "",
+    onClick: () -> Unit = {},
+    onLongClick: () -> Unit = onClick,
```

Comment:
Here:
- I made `linkAnnotationTag` as the string annotation is `""` by default for URL annotations.
- `onClick` can be null if we're only using URL annotations, as they will automatically open their URLs as the default action.
- `onLongClick` behaves the same, but I thought we might want it to default to the same action as `onClick`. I can change it otherwise.

Reply:
> onLongClick behaves the same, but I thought we might want it to default to the same action as onClick. I can change it otherwise.

I would prefer `onLongClick` to have no effect by default. I think it is better UX.